templates/src/prettyprint.c.erb

```diff
@@ -1,6 +1,9 @@
 <%# encoding: ASCII -%>
 #include "prism/prettyprint.h"
 
+#define __STDC_FORMAT_MACROS
```

Comment:
I think this should move to https://github.com/ruby/prism/blob/main/include/prism/defines.h to ensure it's defined before any other system #include (otherwise these macros could be missing and the second include is noop).
I think it makes sense to also move the `#include <inttypes.h>` there.